Networking/Networking/Remote/SiteRemote.swift

```diff
@@ -95,23 +95,29 @@ public class SiteRemote: Remote, SiteRemoteProtocol {
     }
 
     public func uploadStoreProfilerAnswers(siteID: Int64, answers: StoreProfilerAnswers) async throws {
-        let industry: [String]? = {
-            guard let category = answers.category else {
-                return nil
+        let parameters: [String: Any] = {
+            let industry: [String]? = {
+                guard let category = answers.category else {
+                    return nil
+                }
+                return [category]
+            }()
+            let onboarding: [String: Any?] = [
+                "industry": industry,
+                "is_store_country_set": answers.countryCode != nil,
+                "business_choice": answers.sellingStatus?.remoteValue,
+                "selling_platforms": answers.sellingPlatforms
+            ]
+
+            var params: [String: Any?] = [
+                "woocommerce_onboarding_profile": onboarding.compactMapValues { $0 }
+            ]
+            if let countryCode = answers.countryCode {
+                params["woocommerce_default_country"] = countryCode
```

Comment:
Nit: Since we are using `compactMapValues` below, can we get rid of the `if` check and set the nullable default country in the `params` dictionary?

```suggestion
            let params: [String: Any?] = [
                "woocommerce_onboarding_profile": onboarding.compactMapValues { $0 },
                "woocommerce_default_country": answers.countryCode
            ]
```

Reply:
Done in c93e6dfe5613d6e594ea5602dd4199a0c2ff529d